examples/chart/teleport-cluster/templates/auth/deployment.yaml

```diff
@@ -145,6 +145,12 @@ spec:
         - name: diag
           containerPort: 3000
           protocol: TCP
+        - name: auth
+          containerPort: 3025
+          protocol: TCP
+        - name: kube
+          containerPort: 3026
+          protocol: TCP
```

Comment:
Why is there a kube listener exposed on the auth?

Reply:
That's actually a good question. @hugoShaka I think maybe 3026 shouldn't be exposed here? (the `kubernetes_service` listens 'internally' on port 3027 but this isn't exposed to end-users - it's only the proxy part which is exposed)

Reply:
The auth also runs the kube service. This is for legacy reasons and will change at some point. To avoid a circular dependency (proxy connects to auth, but auth kube service connects to proxy) the auth exposes the kube service, and the proxy connects to it directly (instead of doing the usual reverse tunnel stuff and listening on 3027 on loopback).

Reply:
Should we match the `kube` and `k8s` names for 3026?

Reply:
I agree they probably should match.

Reply:
I didn't realize this was automerging 😬

Reply:
#21017